pykafka/protocol.py

```diff
@@ -769,14 +776,28 @@ class FetchResponse(Response):
           HighwaterMarkOffset => int64
           MessageSetSize => int32
     """
-    def __init__(self, buff):
+    api_version = 0
+
+    @staticmethod
+    def get_subclass(broker_protocol):
+        target_version = parse_version(broker_protocol)
```

Comment:
Including a link here to [this documentation](https://cwiki.apache.org/confluence/display/KAFKA/A+Guide+To+The+Kafka+Protocol#AGuideToTheKafkaProtocol-FetchResponse) might help disambiguate this logic for future readers.